### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by FerDR89** [ale-strada/api-m9-vercel#2]
Buenas Ale

Te dejo algunos comentarios:

PATCHme update
{
   "email":"test@mail.com",
   "updateData":{
        "name":"Ale"
   }
}

Le buscaría la vuelta para enviar en el body lo justo y lo necesario para que funcione.
Ej: El email a menos que me permitas cambiarlo desde ahí no lo enviaría (ya me identifico con un token y ese esta asociado a mi userId).
Mismo encerrar la prop name dentro de otro objeto dado que esta claro que este endpoint es para hacer un update de mis datos personales.

Misma situación el endpoint de address con el email.

Lo anterior es una recomendación solamente.

Ahora lo que si no me estaría funcionando es el tema del aviso por email de la confirmación de compra, porfa corregirlo así le puedo dar el visto bueno al desfio.

Éxitos!

**Comment by ale-strada** [comment]
Hola Fer. como va? gracias por los comentarios. El tema del email del comprador, recien termino de arreglarlo. Me di cuenta xq ese aviso me llego a mi, lo habia asociado a mi direccion y no a la del usuario. Acabo de subir el cambio.

**Comment by FerDR89** [comment]
Buenas Ale, 
Probé nuevamente y no me llega el correo.

Fíjate por acá
![image](https://user-images.githubusercontent.com/74626765/199020243-c192d133-2120-44d6-95d2-9ba0780eab16.png)

Creo que te faltan argumentos en esta función

![image](https://user-images.githubusercontent.com/74626765/199020683-b25afbfd-bae6-4c26-b54e-edc7d92dcb09.png)

Y por eso no llega el correo.

**Comment by ale-strada** [comment]
no, no.. lo que pasa ahi es que tengo 2 librerias para mandar email. La que estoy usando es sendgrid es la primera que me mandas sendEmail(msg). La funcion que lleva mas argumentos es para Mailgun. Pero no esta en uso. tengo que borrar ese codigo

**Comment by ale-strada** [comment]
me parece raro que no te llegue, yo lo probe con 2 usuarios distintos y funciono.
